packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -393,6 +393,7 @@ export type CallCompositeIcons = {
 export type CallCompositeOptions = {
     errorBar?: boolean;
     callControls?: boolean | CallControlOptions;
+    permissions?: CallPermissionOptions;
```

Comment:
Naming: this should be more specific. There are many permissions when making a call: ACS scopes, file upload/download if that is supported etc.

See naming discussion for the type below. The field name should follow closely.